api/e_appendix.txt

```diff
@@ -138,20 +143,20 @@ The following features are added to the OpenCL C programming language
   ** *printf* function described in _section 6.12.13_.
   * Storage class specifiers extern and static as described in _section
     6.8_.
-  * Macros `CL_VERSION_1_2` and `+__OPENCL_C_VERSION__+`.
+  * Macros `CL_VERSION_1_2` and `__OPENCL_C_VERSION__`.
 
 The following APIs in OpenCL 1.1 are deprecated (see glossary) in OpenCL
 1.2:
 
-  * *clEnqueueMarker*, *clEnqueueBarrier* and *clEnqueueWaitForEvents*
-  * *clCreateImage2D* and *clCreateImage3D*
-  * *clUnloadCompiler* and *clGetExtensionFunctionAddress*
+  * {clEnqueueMarker}, {clEnqueueBarrier} and {clEnqueueWaitForEvents}
+  * {clCreateImage2D} and {clCreateImage3D}
+  * {clUnloadCompiler} and {clGetExtensionFunctionAddress}
   * *clCreateFromGLTexture2D* and *clCreateFromGLTexture3D*
```

Comment:
Missed `clCreateFromGLTexture2D` and `clCreateFromGLTexture3D`

Reply:
This is actually on purpose, though perhaps I need to think of a better solution.

The reason why this formatting is done "manually" is because these are extension APIs, so my script doesn't add them to the Asciidoc dictionary.  Some options to "fix" this could be:

* Remove these APIs from this section, since they're extension APIs.
* Add the extension APIs to the dictionary, but without a link.

I'll try the second options for now.

Reply:
ahh, thanks for clarifying

Reply:
I've implemented this change - let me know what you think.  Thanks!

Reply:
Looks good, thanks Ben. Does this mean we can update the extension documents to use the API linking too? Not necessarily in this pull request.

Reply:
> Does this mean we can update the extension documents to use the API linking too?

Yes, I think so, although there are a few details to work out if we decided to go this route:

* The API spec is mostly "self-contained", for the most part.  Other than these two items in the appendix, I think the only linking is to other locations in the same doc.

* By contrast, I suspect the extension spec will contain links to the extension spec and to the API spec (and, possibly, to the OpenCL C spec).  I'm not quite sure how to handle this yet...

Reply:
Sounds like this thread is resolved, I was going to open an issue, but seems like this is a continuation of https://github.com/KhronosGroup/OpenCL-Docs/issues/47.